docs/prerequisites/cmake.rst

```diff
@@ -9,3 +9,20 @@ package software.
 
 -  Please download and install the latest binary distribution of CMake
    for your platform from: https://cmake.org/download/
+
+
+.. figure:: ../images/cmake/cmake_release.PNG
+   :alt: Release page of CMake
+   :align: center
+   :width: 600px
+
+   Release Page of CMake
+
+On opening the installer: 
+
+- Open the installer, click NEXT. 
+- Accept the terms and click NEXT.
+- Select the “Add CMake to the system PATH for all users” option in the Install options section. 
+- Click next and finally click Install
+
+After Cmake is installed, restart your system.
```

Comment:
Replace `Cmake` with `CMake`. Add a note if restarting is really necessary (for non technical users)

Reply:
![mj1](https://user-images.githubusercontent.com/66218496/131900765-9f3213df-5882-488e-ae52-2d87ec6ef854.PNG)


Note as in, like this?

Reply:
No no just write it as a part of the instructions itself, so that the audience takes it seriously.

Reply:
Okay okay

Reply:
![mg3](https://user-images.githubusercontent.com/66218496/131903023-18dfbde7-5fc3-497c-af8e-8fd6ba77b8a8.PNG)

Is it fine?

Reply:
Just change NEXT to Next. The rest looks good to go.